src/NServiceBus.Gateway.Tests/App.config

```diff
@@ -24,7 +24,7 @@
     <Channels>
       <Channel Address="http://localhost/headquarter" ChannelType="Http"/>
       <Channel Address="ftp://localhost:21" ChannelType="ftp"/>
-      <Channel Address="https://localhost/headquarter" ChannelType="Http" NumberOfWorkerThreads="3"/>
+      <Channel Address="https://localhost/headquarter" ChannelType="Http" MaxConcurrency="3"/>
```

Comment:
Should we ditch the custom config section and move to a code only api? (could be a different pull)